echopype/convert/set_groups_azfp.py

```diff
@@ -37,6 +37,51 @@ class SetGroupsAZFP(SetGroupsBase):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
+        # get frequency values
+        freq_old = list(self.parser_obj.unpacked_data["frequency"])
+
+        # sort the frequencies in ascending order
+        freq_new = freq_old[:]
+        freq_new.sort(reverse=False)
+
+        # obtain sorted frequency indices
+        self.freq_ind_sorted = [freq_new.index(ch) for ch in freq_old]
+
+        # obtain sorted frequencies
+        self.freq = self.parser_obj.unpacked_data["frequency"][self.freq_ind_sorted]
+
+        # obtain channel_ids
+        self.channel_ids = self._create_unique_channel_name()
```

Comment:
I recommend renaming this to `self.channel_ids_sorted` to that it is clear that this is sorted. Since in the EK60 and EK80 counterpart the same variable name `channel_ids` are not sorted.

Reply:
Good point, I will change this name.